Cannot find module 'tslib'

<!--
Thank you for raising your concerns, we appreciate your feedback and contributions to this repository.

Before you continue, consider the following:

If you have a "How do I do ...?" question, it is better for you and for us that this question is placed in [StackOverflow](http://stackoverflow.com/questions/tagged/rxjs5) or some chat channel. This way, you are making it easier for others to learn from your experiences too.

These "Issues" are meant only for technical problems, bugs, and proposals related to the library.

If your issue is a bug, please follow the format below:
-->

**RxJS version:**
6.0.0-alpha.0
**Code to reproduce:**

**Expected behavior:**

**Actual behavior:**
When I use mocha to run my unit testing, it get this error:
Cannot find module 'tslib'          node_modules/rxjs/Subject.js:3:15
And I find that tslib is not installed.

**Additional information:**
"mocha": "^3.4.2"
"typescript": "2.3.4"
 "typings": "^1.3.2"

--- a/package.json
+++ b/package.json
@@ -199,7 +199,6 @@
     "sinon": "^2.3.5",
     "sinon-chai": "^2.11.0",
     "source-map-support": "^0.4.0",
-    "tslib": "^1.7.1",
     "tslint": "^5.4.3",
     "typescript": "^2.4.0",
     "validate-commit-msg": "^2.3.1",
@@ -212,6 +211,7 @@
   },
   "typings": "./dist/cjs/Rx.d.ts",
   "dependencies": {
-    "symbol-observable": "^1.0.1"
+    "symbol-observable": "^1.0.1",
+    "tslib": "^1.7.1"
   }
 }

